habitat-baselines/habitat_baselines/config/default_structured_configs.py

```diff
@@ -244,7 +244,8 @@ class HrlDefinedSkillConfig(HabitatBaselinesBaseConfig):
 
     # If true, do not call grip_actions automatically when calling high level skills.
     # Do not check either if an arm action necessarily exists.
-    ignore_grip: bool = False
+    # TODO(zxz): ignore grip
+    ignore_grip: bool = True
```

Comment:
I think this is for my new_action: place and reset. If we need the grip action, then there are many more to do with the naming part. That's why I set ignore_grip as True. >_<  I think your statement might be right, since when I modified it, our arm will never automatically pick up objects even if the arm is farrrrr away from the objects.